lms/templates/problem.html

```diff
@@ -5,38 +5,106 @@
 %>
 
 <%namespace name='static' file='static_content.html'/>
-<h3 class="hd hd-2 problem-header">
+<h3 class="hd hd-2 problem-header" id="${ id }-problem-title" aria-describedby="${ id }-problem-progress" tabindex="-1">
```

Comment:
Could we just pass short_id to the template as the id variable instead of changing templates?